packages/themes/blue-jeans/src/index.scss

```diff
@@ -364,6 +364,12 @@
           ),
         ),
         $radio-button: (
+          xs: (
+            border-width: calc(0.2rem * var(--ods-size-ratio)),
```

Comment:
We would prefer a 2px border-with so the radio button will be 16px instead of 18px.
So, border-width: calc(0.125rem * var(--ods-size-ratio))

Reply:
fixed